department-of-veterans-affairs/va.gov-cms
# VACMS-4385: Add featured content to vc.

## Description
See #4385

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/109706422-edb6e880-7b66-11eb-9faf-a99577e862f7.png)

## QA steps
- As an administrator, go to `/node/add/vet_center`
- Visually verify that the new featured content paragraph button appears.
- Add two, and confirm the add more button no longer appears
- Confirm both instances have a call to action button, that when clicked brings up a call to action paragraph within the parent.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
